internal/dnsproviders/gen.py

```
@@ -80,4 +80,18 @@ def unquote(s: str) -> str:
+
+func init() {
+\tregisterPlaceholderProviders(
+"""
+
+with open("../autocert/provider_placeholders_gen.go", "w") as f:
+    f.write(placeholder_header)
+    for name in ["local", "pseudo", *allowlist]:
+        f.write(f'\t\t"{name}",\n')
+    f.write("\t)\n}\n")
+
+os.execvp("gofmt", ["gofmt", "-w", "providers.go", "../autocert/provider_placeholders_gen.go"])
```

Ruff (0.15.12)

[error] Starting a process without a shell

(S606)